src/app/time-picker/time-picker.sample.ts

```diff
@@ -16,11 +16,11 @@ export class TimePickerSampleComponent implements AfterViewInit {
     public max = '19:00';
     public min = '09:00';
 
-    public itemsDelta = { hours: 1, minutes: 5 };
+    public itemsDelta = { hours: 1, minutes: 5, seconds: 1 };
```

Comment:
@jackofdiamond5 @PlamenaMiteva 
Please check the new delta type in refactoring and make sure all props are optional, so just specific parts can be set without failing strict type checks.